documentation/profiles/index.md

```diff
@@ -141,4 +141,4 @@ STIX profiles are generally created through community agreement (for community p
 
 ## Questions?
 
-Profiles are a complicated topic. Please [get in touch](stix-taxii@hq.dhs.gov) with the team if you have any questions or think this documentation could be improved.
+Profiles are a complicated topic. Please [get in touch](mailto:stix-taxii@hq.dhs.gov) with the team if you have any questions or think this documentation could be improved.
```

Comment:
I don't know. The address on home page got changed back to stix@mitre.org at some point, but this didn't get changed. I'll let someone else decide what the link should actually be :smile: